rl/action.py

```diff
@@ -0,0 +1,8 @@
+class Action:
+    def __init__(self, location, connection):
+        self.location = location
+        self.connection = connection
+
+        
+        
+        
```

Comment:
Nitpick: Could remove these blank lines

Reply:
I'll do anything to have the most aesthetic repo in the company

Reply:
We should add `black` as a linter its _amazing_

Reply:
I can get with that